script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt

```diff
@@ -76,52 +82,53 @@ In order to avoid that scenario, Ciel turns herself into a puppet.
 [sha:ca7964a8f017c948759fd8d8a8a10677935729a7]{
 -- Page 263, Offset 32391.
 -- 自らが定めた『命令』だけを行う、何も考えない、何ごとにも動じない、自動人形になったのである。
-She turns herself into an automata that follows her own pre-determined path, without thought, unshaken by her surrounding.
+An automaton that only follows the orders she's lain out in advance, unthinking, unswerving.
 }
 [sha:9d6c030ef6daeddd01ef9ed586de1f08272d9328]{
 -- Page 264, Offset 32392.
 -- シエルは自動的に、ロアの死角になる確殺地点に移動する。
-Automatically, Ciel moves toward Roa's blind spot that will become the place of his certain death.
+Ciel mechanically advances on the blind spot that will spell Roa's inevitable death.
 }
 [sha:5630be87ba37db84f2a112f43efcdfb3c0f80edc]{
 -- Page 264, Offset 32393.
 -- 周囲の情報は感知魔術によって把握していた。
-She understands the information of her surroundings through Perception Magecraft.
+Through Perception Magecraft she can sense what's going on around her.
 }
 [sha:9a571803ab01423f50d19b12bb196feea538f5f0]{
 -- Page 265, Offset 32394.
 -- ロアは体育館に多数の魔術式を敷いて待ち構えている。
-Roa has laid out a great number of magical formulas, lying in wait in the gymnasium.
+Roa lies in wait in the gym, surrounded by a vast number of magical formulas he's inscribed.
 }
 [sha:0564373c8d8349502ae9d9044253add6783ad12b]{
 -- Page 265, Offset 32395.
 -- アルクェイドは渡り廊下を体育館に向かって移動している。
-Arcueid is moving toward the gymnasium through the connecting hallway.
+Arcueid is moving toward the gym through the connecting hallway.
 }
 [sha:9ad63e4a33b02774ef95bd220bd7ad93b8db6b64]{
 -- Page 265, Offset 32396.
 -- ちりちりとコンクリートの表面が焦げている。
-The concrete surface sparks and scorches.
+Sparks crackle across the concrete face of the building.
 }
 [sha:7f2b4a0503846bbea0b1ab8d39890e9dc687ab4a]{
 -- Page 266, Offset 32397.
 -- ロアがここまで貯蔵した<魔|オ><力|ド>と、
-The <magical energy|o d   > that Roa preserved until now,
+The <magical energy|o d    > that Roa has been gathering up until now―――
 }
 [sha:56b0bf6f1be606d72298fc24805698797bc0a76c]{
 -- Page 266, Offset 32398.
 -- アルクェイドの持つ魔力は拮抗している。
-contends with Arcueid's own magical energy.
+is jockeying for supremacy with Arcueid's own magical energy.
 }
 [sha:b4c0976a0c4059822e66f1769c38ddc8e586cc6e]{
 -- Page 266, Offset 32399.
 -- シエルに意識があれば火薬庫を連想しただろう。
-If Ciel had been conscious, it would have reminded her of an ammunition dump.
+If Ciel had been conscious, it would have reminded her of a powder keg.
```

Comment:
I don't really like "conscious" but i can't think of any alternatives off the top of my head. As I understand it, shes unconscious in that she's acting on instinct, not in that she's asleep.